DirectProgramming/DPC++FPGA/Tutorials/DesignPatterns/compute_units/src/compute_units.cpp

```diff
@@ -0,0 +1,109 @@
+//==============================================================
+// Copyright Intel Corporation
+//
+// SPDX-License-Identifier: MIT
+// =============================================================
+#include <CL/sycl.hpp>
+#include <iostream>
+// dpc_common.hpp can be found in the dev-utilities include folder,
```

Comment:
add a space here to separate the includes

Reply:
Ok, I added a blank line before the comment.